apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -432,35 +511,32 @@ export class CaseService {
       },
     ]
 
-    // Not modifying the ruling
-    if (!theCase.rulingDate) {
-      const deliverCaseFileToCourtMessages =
-        theCase.caseFiles
-          ?.filter(
-            (caseFile) =>
-              caseFile.state === CaseFileState.STORED_IN_RVG && caseFile.key,
-          )
-          .map((caseFile) => ({
-            type: MessageType.DELIVER_CASE_FILE_TO_COURT,
-            userId: user.id,
-            caseId: theCase.id,
-            caseFileId: caseFile.id,
-          })) ?? []
+    const deliverCaseFileToCourtMessages =
```

Comment:
We no longer care is we are modifying or not. We always want to update external systems.